connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/expressions/LiteralProtoConverter.scala

```diff
@@ -0,0 +1,289 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.spark.sql.expressions
+
+import java.lang.{Boolean => JavaBoolean, Byte => JavaByte, Character => JavaChar, Double => JavaDouble, Float => JavaFloat, Integer => JavaInteger, Long => JavaLong, Short => JavaShort}
```

Comment:
searched on the codebase, Spark used `import java.lang.{Boolean => JBoolean, Integer => JInteger, Long => JLong}` in other places.

Reply:
not all

https://github.com/apache/spark/blob/d7d8af0dbb47e152b280226a7afcf0771b5a5ae8/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/literals.scala#L20-L31

Reply:
> searched on the codebase, Spark used `import java.lang.{Boolean => JBoolean, Integer => JInteger, Long => JLong}` in other places.

but let me follow this

Reply:
done